keyboards/tanuki/Makefile

```diff
@@ -0,0 +1,5 @@
+
```

Comment:
This file should be removed.  It's not needed, and the recommended method for compilation is from the QMK root folder, using `make keyboard:keymap:flasher`